doc-site/docs/components/kubernetes/notification-service.md

```diff
@@ -13,8 +13,24 @@ sidebar_position: 5
 - SMS capabilities
   - twillio
 
+## REST API
+The API is defined using OpenAPI 3.0 and you explore the service details by
+- viewing the [API specs yaml file][notification-api-specs] or
+- with any compatible tools such as [Swagger Editor][browse-api-spec]
+
+## Configuring
+By default Zero uses helm to bundle the service, you just need to enable the service and provide necessary API keys and it will work out of the box.
+### Available values
+[See helm chart][notification-available-values] for all the available values, under `application` you can set up your API keys and application related parameters.
+### Setting up API keys
+API keys will be created as a Kubernetes secret. It is recommended you use `set_sensitive` in terraform similar to [this example][eks-set-sensitive-example]
```

Comment:
This is not ideal, people shouldn't really need to add `set_sensitive` options to the terraform, the secrets should already exist. We're sort of working around the fact that twilio support wasn't added properly here.